### PR TITLE
ISSUE-1: Adjust styles for webforms, pages, content listings

### PR DESCRIPTION
**Pull request by marlo-longley**
Addresses #1 (General Styling issue)

What is new?

- Add border/divider below webform

- Make buttons bigger and stronger colors

- Align pages content to edge of gray top navbar

- Remove custom styles from radios and checkbox. This means default Bootstrap styles for now. Can customize at later time if desired.

- Add better style for pager results summary at top of list (class views-header) (this means yes, I enabled pagers thru the Drupal UI on server)

- Change highlight from white to orange on main content nav tabs

- Add darker gray background to Moderation State portion outside of webform, and also filters and form elements above content listings

- Make items inside dropdown nav menus expand automatically (aka not nest and remove dropdown header/category like "Create") when in mobile/hamburger menu

- Align filters above content listings

@DiegoPino 

(will address JS for admin toolbar in different pull)

### CODE REVIEW

File: css/custom.css [hunk 1]
```diff
@@ -48,6 +57,36 @@ main .feed-icons {
...
+    padding: 15px 0;
+    background-color: rgba(0, 0, 0, 0.063);
+    -moz-border-radius: 8px;
+    -webkit-border-radius: 8px;
+    border-radius: 8px; /* future proofing */
+    -khtml-border-radius: 8px; /* for old Konqueror browsers */
+}
+
+/* This is the "Apply to selected items" button in the filter area above content listings */
+#edit-submit--2 {
```
Comment (DiegoPino) [inline css/custom.css:86]: Is this selector "user changes" aware? Any way you can target this without relaying on Drupal's weird-o auto naming? Just a question, not a change request

File: css/form.css [hunk 2]
```diff
@@ -1,15 +1,31 @@
-/* Node edit buttons */
-#edit-submit {
-    background-color: rgba(191, 213, 193, 1)
+/* Border under main webform */
+#edit-field-descriptive-metadata-wrapper {
+    border-bottom: 1px #656565 solid;
+}
+/* Buttons */
+#edit-submit, #edit-submit--2, .form-submit {
```
Comment (DiegoPino) [inline css/form.css:6]: Is this targeting same html id as https://github.com/mnylc/esie_subtheme/pull/9/files#diff-6a1817805a1dbb597d50598859417457R86 ?

File: css/form.css [hunk 3]
```diff
@@ -109,4 +124,19 @@
 
 .js fieldset.form-type-search-api-autocomplete input.form-autocomplete.ui-autocomplete-loading:focus {
     background-image: url(../../../../core/misc/throbber-active.gif);
-}
\ No newline at end of file
+}
+
+/* This is the "Current state: Draft" etc. section below webforms */
```
Comment (DiegoPino) [inline css/form.css:129]: good!

File: css/navbar.css [hunk 4]
```diff
@@ -9,9 +9,50 @@ header > nav:first-child {
     background-position: 80% 0%;
 }
 
-#CollapsingNavbar > div:first-child {
-    display: flex;
-    justify-content: flex-end;
+
+@media (min-width: 993px) {
```
Comment (DiegoPino) [inline css/navbar.css:13]: Just a question, never seen selectors with a single pixel difference between max-width and min-width. Is that normal or a typo?

File: css/webform.css [hunk 5]
```diff
@@ -50,15 +50,17 @@
...
-.webform-actions .webform-button--next {
-    background-color: #bfd5c1;
+.webform-actions .webform-button--next, .webform-actions webform-button--submit, button[name="geographic_location_reconciliate_button"] {
+    background-color: rgba(31, 111, 108, 137);
+    color: white;
 }
 
-.webform-button--submit {
-    background-color: rgb(189, 213, 173);
+button[name="images_remove_button"] {
```
Comment (DiegoPino) [inline css/webform.css:58]: nice!

File: templates/form/input--submit-button.html.twig [hunk 6]
```diff
@@ -12,4 +12,4 @@
  * @see template_preprocess_input()
  */
 #}
-<button{{ attributes }}>{{ attributes.value|raw }}</button>{{ children }}
\ No newline at end of file
+<button{{ attributes.removeClass('btn-sm') }}>{{ attributes.value|raw }}</button>{{ children }}
```
Comment (DiegoPino) [inline templates/form/input--submit-button.html.twig:15]: Can the btn-sm /lg be set on the Barrio UI too? Or is that fixed?

**Comment by DiegoPino** [comment]
@marlo-longley i will merge, but left some questions for you in the code. Thanks

File: css/navbar.css (hunk 4 above)
Comment (marlo-longley) [reply css/navbar.css:13]: @DiegoPino This is normal in my experience (difference of a pixel when using min-width vs. max-width). Min-width being the best practice, mobile-first approach, but sometimes using max-width in a query is necessary.

Ideally you could use SASS mixins for Bootstrap media queries like `@include media-breakpoint-up(sm) { ... }` which deals with that pixel overlap and is less error prone but this assignment was just CSS.

See https://getbootstrap.com/docs/4.0/layout/overview/#responsive-breakpoints

File: templates/form/input--submit-button.html.twig (hunk 6 above)
Comment (marlo-longley) [reply templates/form/input--submit-button.html.twig:15]: @DiegoPino not sure your question. I am not touching the barrio theme code at all.

File: css/form.css (hunk 2 above)
Comment (marlo-longley) [reply css/form.css:6]: @DiegoPino yes you are right. I could group in a better way in same file. Assuming there will be more edits, I can push that then.

**Pull request merged**